src/test/java/io/github/sskorol/testcases/AmazonTests.java

```diff
@@ -110,8 +109,8 @@ public void shouldSearchForShoes(final Shoes shoes, final Account account) {
                 .selectProduct()
                 .buy();
 
-        customAssertThat(shoes)
-                .hasPurchaseStatus(at(ProductPage.class).getPurchaseStatus());
+        customAssertThat(at(ProductPage.class))
+                .hasPurchaseStatus(PurchaseStatus.SUCCESSFUL.getPurchaseStatus());
```

Comment:
if we use one enum for statuses we will use the more obvious name for getter and static import
`LOGIN_SUCCESSFUL`/`PURCHASE_FAILED`.
this line would look so:
`.hasPurchaseStatus(PURCHASE_SUCCESSFUL.getStatus());`